api/docs.go

```diff
@@ -76,6 +76,18 @@ const docTemplate = `{
                         "description": "Search term for name and url.",
                         "name": "search",
                         "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Filter to repository that matches name exactly",
+                        "name": "name",
+                        "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Filter to repository that matches url exactly",
```

Comment:
ditto to the above.

Reply:
repositories ? More than one is being filtered to find the exact match.